api/v1alpha1/repository.go

```diff
@@ -58,3 +59,11 @@ func GetImageOverridePath() []string {
 	}
 	return strings.Split(v, ":")
 }
+
+// RepositorySource is the value of annotation repository.kubebb.k8s.com.cn/source
+var RepositorySource = map[string]string{
```

Comment:
@dayuy pls refer to [here](https://poe.com/s/eaW9cGMWoIQZsYaW0ajL) to define a enum